examples/MicroserviceExample/WorkerService/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OpenTelemetry.Trace;
+
+namespace WorkerService
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureServices((hostContext, services) =>
+                {
+                    services.AddHostedService<RabbitMqConsumer>();
+
+                    // TODO: Determine if this can be done here in a WorkerService. It does not seem to work... doing this in the RabbitMqConsumer for now.
```

Comment:
The `TODO` is really interesting! It should be here right? Wonder if we have a bug somewhere?

Reply:
Yea, I'll circle back in a bit and see what's going on here.